src/script/calling/CallingRepository.ts

```diff
@@ -571,11 +585,12 @@ export class CallingRepository {
       }
       case CALL_MESSAGE_TYPE.CONFKEY: {
         if (source !== EventRepository.SOURCE.STREAM) {
-          const {id, domain} = conversationId;
-          const allClients = await this.core.service!.conversation.getAllParticipantsClients(id, domain);
-          // We warn the message repository that a mismatch has happened outside of its lifecycle (eventually triggering a conversation degradation)
+          const allClients = await this.clientRepository.getQualifiedClientsByUserIds(
+            conversationEntity.participating_user_ids(),
+            false,
+          );
           const shouldContinue = await this.messageRepository.updateMissingClients(
```

Comment:
```suggestion
          // We warn the message repository that a mismatch has happened outside of its lifecycle (eventually triggering a conversation degradation)
          const shouldContinue = await this.messageRepository.updateMissingClients(
```

please don't remove useful comments